### PR TITLE
WRN-11218: Added `@babel/plugin-proposal-class-properties` plugin to fix sampler build

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
While webos building in RP, build errors occurred in com.webos.app.enactsandstonesampler
> ERR! => Failed to build the preview
ERR! ./node_modules/@enact/core/kind/kind.js 108:263
ERR! Module parse failed: Unexpected token (108:263)
ERR! File was processed with these loaders:
ERR!  * ./node_modules/@enact/storybook-utils/node_modules/babel-loader/lib/index.js
ERR! You may need an additional loader to handle the result of these loaders.
ERR! | render,styles:cfgStyles}=config;const renderStyles=cfgStyles?styles(cfgStyles):false;const renderComputed=cfgComputed?computed(cfgComputed):false;const renderKind=(props,context)=>{if(renderStyles)props=renderStyles(props,context);if(renderComputed)props=renderComputed(props,context);return render(props,context);};let Component;// In 4.x, this branch will become the only supported version and the class branch will be
ERR! | // removed.
ERR! > if(functional){Component=function(props){const ctx=useContext(contextType);const boundHandlers=useHandlers(handlers,props,ctx);const merged={...props,...boundHandlers};return renderKind(merged,ctx);};}else{Component=class extends ReactComponent{static contextType=contextType;constructor(){super();this.handlers=new Handlers(handlers);}render(){this.handlers.setContext(this.props,this.context);const merged={...this.props,...this.handlers.handlers};return renderKind(merged,this.context);}};}if(name)Component.displayName=name;if(propTypes)Component.propTypes=propTypes;if(defaultProps)Component.defaultProps=defaultProps;// Decorate the Component with the computed property object in DEV for easier testability
ERR! | if(process.env.NODE_ENV!=="production"&&cfgComputed)Component.computed=cfgComputed;const defaultPropKeys=defaultProps?Object.keys(defaultProps):null;const handlerKeys=handlers?Object.keys(handlers):null;Component.inline=(props,context)=>{let updated={...props};if(defaultPropKeys&&defaultPropKeys.length>0){defaultPropKeys.forEach(key=>{// eslint-disable-next-line no-undefined
ERR! | if(props==null||props[key]===undefined){updated[key]=defaultProps[key];}});}if(handlerKeys&&handlerKeys.length>0){// generate a handler with a clone of updated to ensure each handler receives the same
ERR!  @ ./stories/default/About.js 4:169-204 4:933-937
ERR!  @ ./stories/default sync nonrecursive ^\.\/(?:(?!\.)(?=.)[^/]*?\.js)$
ERR!  @ ./.storybook/generated-stories-entry.js
ERR!  @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js ./.storybook/storybook-init-framework-entry.js ./.storybook/preview.js-generated-config-entry.js ./.storybook/generated-stories-entry.js
....

### Resolution
Added `@babel/plugin-proposal-class-properties` plugin we've removed from https://github.com/enactjs/storybook-utils/pull/14.

### Additional Considerations
In https://github.com/enactjs/storybook-utils/pull/14, we've removed dependency also, but `@babel/preset-env` has the dependency so we don't need to add it again.

### Links
WRN-11218

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
Please add a changelog

File: configs/babel.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ module.exports = function (api) {
 
 			// Stage 3
 			// '@babel/plugin-syntax-import-meta',
+			[require('@babel/plugin-proposal-class-properties').default, {loose: true}],
```
Comment (MikyungKim) [inline configs/babel.js:85]: Should we add `@babel/plugin-proposal-class-properties` dependency to package.json?

Comment (hong6316) [reply configs/babel.js:85]: Don't need to.
It's already included in `@babel/preset-env` so we can use without adding in package.json

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# unreleased
+
+* Added `@babel/plugin-proposal-class-properties` plugin to build sandstone sampler.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* Added `@babel/plugin-proposal-class-properties` plugin to fix sampler build.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# unreleased
```
Comment (MikyungKim) [inline CHANGELOG.md:1]:
```suggestion
## unreleased
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**